### PR TITLE
Build page on /automotive/developers

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build page on /automotive/developers

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Demo at https://ubuntu-com-12022.demos.haus/automotive/developers
- Compare page against [copy doc](https://docs.google.com/document/d/1tO6WLB0Mk4jx_s22Yi2502nfGzsXbuNZCzSr56MnHWw/edit#) 

## Issue / Card

Fixes [#5933](https://github.com/canonical/web-squad/issues/5933)

### CODE REVIEW

File: templates/automotive/developers.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          height="120",
+          width="198",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <p>Get trusted provenance for open-source software.</p>
+      <p>Confidently integrate open-source components into your proprietary solutions.</p>
+      <p><a href="/support">Simplify compliance. Streamline certification. Protect innovation.</a></p>
```
Comment (mtruj013) [inline templates/automotive/developers.html:127]:
```suggestion
      <p><a href="/support">Learn more about support services&nbsp;&rsaquo;</a></p>
```

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Thanks @minkyngkm, looks great! A couple notes, we made some updates to some of the links in the doc which weren't descriptive enough for users with screen readers. There's also supposed to be a CTA at the top of the page according to the doc.

**Comment by minkyngkm** [comment]
Thanks @mtruj013 - Nice suggestions!! I've made some changes :)

**Review by mtruj013 — APPROVED** [review]
LGTM!

**Pull request merged**